nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/main/java/org/nuxeo/elasticsearch/audit/ESAuditBackend.java

```diff
@@ -121,6 +121,13 @@ public ESAuditBackend(NXAuditEventsService component, AuditBackendDescriptor con
         super(component, config);
     }
 
+    /**
+     * @since 9.3
+     */
+    public ESAuditBackend() {
+        super();
```

Comment:
The parent class has a constructor `AbstractAuditBackend(NXAuditEventsService component, AuditBackendDescriptor config)`  Are component and config not needed or injected some other way?

Reply:
I needed a constructor when your are using this backend as just an audit storage. In this case, those fields are not necessary.